packages/massa-web3/src/experimental/basicElements/address.ts

```diff
@@ -43,58 +46,101 @@ export class Address {
    *
    * @throws If the version is not supported.
    */
-  protected static initFromVersion(version: Version = Version.V0): Address {
+  protected static initFromVersion(
+    version: Version = DEFAULT_VERSION
+  ): Address {
     switch (version) {
       case Version.V0:
-        return new Address(new Base58(), new VarintVersioner(), version, false)
+        return new Address(new Base58(), new VarintVersioner(), version)
       default:
         throw new Error(`unsupported version: ${version}`)
     }
   }
 
+  private getPrefix(str: string): string {
```

Comment:
Should this function be changed to static since we don't use 'this'? What is the best practice in TypeScript?

Reply:
i am not aware of any best practice about this. lets go for static then it should be totally equivalent